src/custom-typings.d.ts

```diff
@@ -0,0 +1,10 @@
+/*
+  This file is to fix a missing type from styled-components.
+  It can be deleted when https://github.com/styled-components/styled-components/pull/1827 gets released
+*/
+import * as React from "react";
+import { Component } from "react";
+
+declare module "styled-components" {
+   export function isStyledComponent(target: string | Component<object>): boolean;
+}
```

Comment:
Not crazy about this file name, it should probably be called `styled-components.d.ts`. I'm also not crazy about putting in directly in the `src` folder. 

Should look at moving it to a [`typings` folder](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types). 

Maybe something like `typings/npm/styled-components/index.d.ts`